scripts/docs.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
+set -u
+
+if [ -z `which jazzy` ]; then
+    echo "Installing jazzy…"
+    gem install jazzy
+    if [ -z `which jazzy` ]; then
+        echo "Unable to install jazzy."
+        exit 1
+    fi
+fi
+
+for module in MapboxNavigation MapboxCoreNavigation
+do
+    jazzy \
+        --config jazzy.yml \
+        --module "$module" \
+        --module-version 0.4.0 \
+        --root-url 'https://www.mapbox.com/navigation-sdk/ios/0.4.0/' \
```

Comment:
`root-url` tells jazzy where the documentation will wind up. The documentation’s location is determined by where you upload it to. In this case, this PR would have used GitHub Pages, which would have made the documentation accessible from mapbox.github.io, not mapbox.com.